docs/reference/esql/functions/st_contains.asciidoc

```diff
@@ -0,0 +1,34 @@
+[discrete]
+[[esql-st_contains]]
+=== `ST_CONTAINS`
+
+*Syntax*
+
+[.text-center]
+image::esql/functions/signature/st_contains.svg[Embedded,opts=inline]
+
+*Parameters*
+
+`geomA`::
+Expression of type `geo_point`, `cartesian_point`, `geo_shape` or `cartesian_shape`. If `null`, the function returns `null`.
+
+`geomB`::
+Expression of type `geo_point`, `cartesian_point`, `geo_shape` or `cartesian_shape`. If `null`, the function returns `null`.
+The second parameter must also have the same coordinate system as the first.
+This means it is not possible to combine `geo_*` and `cartesian_*` parameters.
+
+include::description/st_contains.asciidoc[]
+This is the inverse of the `<<esql-st_within,ST_WITHIN>>` function.
+
+include::types/st_contains.asciidoc[]
+
+*Example*
+
+[source.merge.styled,esql]
+----
+include::{esql-specs}/spatial.csv-spec[tag=st_contains-airport_city_boundaries]
+----
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+include::{esql-specs}/spatial.csv-spec[tag=st_contains-airport_city_boundaries-results]
+|===
```

Comment:
You can *generally* just link to `esql/functions/layout/st_contains.asciidoc` - especially if you add an `@Example` to `@FunctionInfo` annotation. Then it *should*  spit out pretty much the same thing as this. You may want a `note` for the "The second parameter must al..." part.

Reply:
Cool! I'm trying the @Example option, but will leave the rest of the options alone in this PR and play with this more in later work.